packages/ember-views/tests/views/view/init_test.js

```diff
@@ -30,7 +30,7 @@ test("registers view in the global views hash using layerId for event targeted",
 
 QUnit.module("EmberView.createWithMixins");
 
-test("should warn if a non-array is used for classNames", function() {
+test("should warn if a computed property is used for classNames", function() {
```

Comment:
> I can do a separate PR to test that classNames is actually an array.

Can you clarify this? There must be other tests for `classNames` that use an array already, yes?